robotiq_c_model_control/src/robotiq_c_model_control/attachment_session.py

```diff
@@ -0,0 +1,23 @@
+"""
+Copyright 2018 by READY Robotics Corporation.
+All rights reserved. No person may copy, distribute, publicly display, create derivative works from or otherwise
+use or modify this software without first obtaining a license from the READY Robotics Corporation.
+"""
+from bondpy import bondpy
+
+
+class AttachmentSession(object):
```

Comment:
I love this file. You may want to consider moving it to the ros_shared_library

Reply:
I'll make an issue to do that.

Reply:
Shared library_python as opposed to _common I assume?